util/argParser.js

```diff
@@ -68,12 +68,19 @@ class ArgParser {
         type: "number"
       },
 
-      "limit": {
+      "pageLimit": {
+        alias: "limit",
         describe: "Limit crawl to this number of pages",
         default: 0,
         type: "number",
       },
 
+      "maxPageLimit": {
+        describe: "maximum pages to crawl, overriding pageLimit if both are set",
```

Comment:
```suggestion
        describe: "Maximum pages to crawl, overriding pageLimit if both are set",
```

Bit of a nit, but the other arg descriptions start with a capital letter